README.md

```diff
@@ -27,11 +28,17 @@ for Java, with the following changes (mandatory):
 
 * IntelliJ
   - Go to "Preferences" and them to "Coding Style" in IntelliJ.
-  - Inport [intellij-java-magicdgs-style.xml](https://github.com/magicDGS/styleguide/blob/master/intellij-java-magicdgs-style.xml)
+  - Import [intellij-java-magicdgs-style.xml](https://github.com/magicDGS/styleguide/blob/master/intellij-java-magicdgs-style.xml)
+
+* Eclipse
```

Comment:
IntelliJ have an option to import Eclipse styles. Did you check if this ones are working as expected if you do that?